src/components/Project/ImageAnalysis/ImageViewer.vue

```diff
@@ -164,6 +130,17 @@ onMounted(() => {
     })
   })
 
+  // Get coordinates of the line
+  image.on('pm:create', (e) => {
+    // Save last drawn line
+    lastDrawnLine = e.layer
+    e.layer.on('pm:edit', handleEdit)
+    findPixelCoords(e.layer.getLatLngs())
+  })
+}
+
+// Callback function for when a line is edited and checks if it's within bounds
+function handleEdit(event) {
   // Checks if the point is within the bounds of the image
   function isWithinBounds(point) {
```

Comment:
Is there a reason we need this function within another function?

Reply:
moved it